### ISSUE TITLE
 Meine Einsätze (memberjobs) - Do not show assignments or show it different that belong to a job with 'Arbeitseinsatz vielfaches'=0h

### ISSUE THREAD

**Issue by padili** [juntagrico/juntagrico#553]
**Problem**
Improvement depends on 'Meine Einsätze' - template memberjobs
We use jobs additionally as a subscription tool:
![image](https://user-images.githubusercontent.com/25586157/190902646-9be97a68-9ac9-47ed-9fed-7ae731e299d4.png)

The 'subscription' job is configured with 'Arbeitseinsatz vielfaches' = 0h (see screenshot).
![image](https://user-images.githubusercontent.com/25586157/190903076-8090a8b9-ac8d-4b44-835d-5f01060f5b24.png)

At the moment the assignments created for the subscription are shown in Meine Einsätze.
This is quite confusing for the members as they have two entries for eg. 1h work in the garden (1x for the subscription, 1x for the assignment-request) that are shown in the same way.

**Expected**
there are different possibilities to solve the problem.
- do not show assignments of jobs with  'Arbeitseinsatz vielfaches'=0h > eventually not suitable for other Solawis?
- have a switch on a job as subscription only. Be aware the assignment is still needed to see who subscribed.

> don't show subscriptions or show it on another way to be clear its only a subscription. i would suggest to do two section on memberjobs like:
![image](https://user-images.githubusercontent.com/25586157/190904313-e3d2eda2-c4a4-41d5-b291-a88bc8e8e7da.png)

**Refinement**

**Todo**

**Comment by pascalfree** [comment]
I agree that these kind of signups should be displayed differently than normal jobs in all places they appear.

At the moment we are abusing the jobs for something they are not (but almost). If we want to disconnect it completely (and show it in a separate list) we should consider a new entitiy (e.g. "events") for it.

Pragmatically I would keep it in the same list but make it e.g. italic and a bit transparent.

In you specific case I would go further and make the "job-extras" feature work to support that: when signing up to the actual job members would select, if the want to join lunch or not.
